events.md

```diff
@@ -28,3 +28,11 @@ The event types to which you can subscribe are shown below.
 | Terminated | epayments.payment.terminated.v1 |
 
 See [ePayment Webhooks](https://developer.vippsmobilepay.com/docs/APIs/epayment-api/features/webhooks) for the payload format.
+
+## Qr
+
+| Name       | Event Type                      |
+| ---------- | ------------------------------- |
+| Scanned    | user.checkin.v1                 |
```

Comment:
I think the name should be CheckIn instead of Scanned. The reason being, that in the future there might come a CheckOut as well